gazebo_plugins/include/gazebo_plugins/gazebo_ros_camera.hpp

```diff
@@ -162,6 +162,15 @@ class GazeboRosCamera
   /// param[in] _enabled True to enable.
   void SetCameraEnabled(const bool _enabled);
 
+  // Get pointer to the cameras
+  const std::vector<gazebo::rendering::CameraPtr> GetCameras() const;
```

Comment:
cpplint wants an `#include <vector>` at the top of this file due to this addition

Reply:
Added